src/shared/utils.ts

```diff
@@ -450,119 +346,21 @@ export function getLanguage(override?: string): string {
       : "browser");
 
   if (lang == "browser" && isBrowser()) {
-    return getBrowserLanguage();
+    return getBrowserLanguages();
   } else {
-    return lang;
+    return [lang];
   }
 }
```

Comment:
Makes more sense to return a list of preferred languages here, momentjs uses this. Unfortunately i18next doesnt, it can only set a single language. So if a word isnt translated into the browser language, it will always fallback to English, even if the user specified another language as second priority.